src/status_im/wallet/choose_recipient/core.cljs

```diff
@@ -93,29 +101,36 @@
 (fx/defn fill-request-from-url
   {:events [:wallet/fill-request-from-url]}
   [{{:networks/keys [current-network] :wallet/keys [all-tokens] :as db} :db} data origin]
-  (let [current-chain-id                       (get-in constants/default-networks [current-network :config :NetworkId])
-        {:keys [address chain-id] :as details} (extract-details data current-chain-id all-tokens)
-        valid-network?                         (boolean (= current-chain-id chain-id))
-        previous-state                         (get-in db [:wallet :send-transaction])
-        old-symbol                             (:symbol previous-state)
-        new-symbol                             (:symbol details)
-        old-amount                             (:amount previous-state)
-        new-amount                             (:value details)
-        new-gas                                (:gas details)
-        symbol-changed?                        (and old-symbol new-symbol (not= old-symbol new-symbol))]
-    (cond-> {:db db}
-      (not= :deep-link origin) (assoc :dispatch [:navigate-back]) ;; Only navigate-back when called from within wallet
-      (and address valid-network?) (update :db #(fill-request-details % details false))
-      symbol-changed? (changed-asset old-symbol new-symbol)
-      (and old-amount new-amount (not= old-amount new-amount)) (changed-amount-warning old-amount new-amount)
+  (assoc-in db [:wallet :eip681] nil)
+  (if (and (string/includes? data "transfer")
+           (< (string/index-of data ".eth") (string/index-of data "transfer")))
+    {:resolve-address
+     {:registry  (get ens/ens-registries (ethereum/chain-id->chain-keyword (get-in constants/default-networks [current-network :config :NetworkId])))
+      :ens-name  (get-in (eip681/parse-uri data) [:address])
+      :cb        #(re-frame/dispatch [:wallet.send/resolve-token-address-from-qr %])}}
```

Comment:
you just need to pass the data here like this:

```suggestion
      :cb        #(re-frame/dispatch [:wallet.send/resolve-token-address-from-qr data %])}}
```

Reply:
This fixed it.  I had been trying to pass data after the anonymous parameter and that's what kept messing me up.

Reply:
I went ahead and pushed a new commit that resolves the hardcoding issue I was having.  Thanks for pointing out that easy remedy to my problem!